

async function main() {
  const address = '0xf62e08643635C0e0755CE5A894fDaEEEF72f8F00';
  const Box = await ethers.getContractFactory('PlonkVerifier');
  const box = await Box.attach(address);
  console.log(process.argv);
  const value = await box.verifyProof("0x118a5e998d767a2fc91060d6836200975bba67ce88382000c6e2bf0a6553ccdd18dda046f20412a1275dfa7b14963e48fa9f378bd9a00895653fa44ebbd9cc8305c8be2214bf2e6a49f9654008f887c534bb5c28041784d7c5c3ec90440a252003f88e9de40242f8705a8872fd342818ad5247cfe63928d92973315955837a2203ccc628d374d25523a256c7b8b72930395668cff1443ef84626bfef34dd9d190e6a9bf2c6e3f7cc57bfb4f7cbbabf2990e7203621ba19fb55f5a7ffe62ffc1c1e6541d1c70ff614759f6cc679094f94d1b579a23abefa7d4e7fc3c25915e8dd176e070976277d3765e5ceb9872bf483b9bd231aed22678b2698c6e2020d53081d3aab6e72b5c6a6234c74345a28dba78c44392ef86c3de75c38b9ce0c038ff1035d4d4742d73dfdf8c89550953b55e8843e86905761f6fd027da81dd2fb387d00f810208f0f8f402e93feec5e53d355cd5d814971ef9f9f3f9048ea0fec05b42e73eb549d9fd29801868636afc38a3c503be183732c156f5a839cc7fe0c616323fcd892f85cad7f9dbd6adcd790483d359a86587aa015b344614763b647836014a0f3e7fd8e6bbe3b3d627c55fa31fed09032faf665d3b74464d818a4f2f2480a8ccf1a6c2761a8d1717726c5660563acd9e697875537e15febe08e026b23912306b395736331b9fb1bc30cf84779f8ce7068f0895d859a66b12bb1c7bcf68205051b63daf2d179575a7ab057bee57337cc789fd4b6cd23ddcbecd15501b7fb26aa91c4511c12eaa14fb21c88018ba11c7d38e3202081186b824634f8bfbcac017dd9fdbb21d8cea822dae18593eafcb7b0526d8654486933c4db689118e7592ca37401e1daea9f94ba55798db4d76c0a3f51b0709f92edd806114fd8d9f0641eb2076321bf3f9c9abfc314168f551a2d74a32a949ca6e14fcc79137dcd962413339b00bee22a7d2892144a09669f9d98400247e288b58a4189c402d95046ef03fcbfcbaf0a8411ab6c1459841831f8442dcedf2f6bb1a7f8da8e9db5f5acb51782fa16f9ca9219920829632b69af618bc721a1a4c8538c679535655b755419222dc4ea0eb1e1ebf03324a0bd6eb1db408e35a3731441af335737489122a5a0",["0x0000000000000000000000000000000000000000000000000000000000000001","0x0000000000000000000000000000000000000000000000000000000000000012"]);
  console.log(value);
}

main()
 .then(() => process.exit(0))
 .catch(error => {
   console.error(error);
   process.exit(1);
 });